Configuration/ProcessModifiers/python/trackingMkFit_cff.py

```diff
@@ -1,4 +1,26 @@
 import FWCore.ParameterSet.Config as cms
 
-# This modifier sets replaces the default pattern recognition with mkFit (possibly in selected iterations only)
-trackingMkFit = cms.Modifier()
+from Configuration.ProcessModifiers.trackingMkFitCommon_cff import *
+from Configuration.ProcessModifiers.trackingMkFitInitialStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitLowPtQuadStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitHighPtTripletStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitLowPtTripletStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitDetachedQuadStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitDetachedTripletStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitMixedTripletStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitPixelLessStep_cff import *
+from Configuration.ProcessModifiers.trackingMkFitTobTecStep_cff import *
+
```

Comment:
why did you choose to make use of individual modifiers ?

Reply:
Individual Modifiers allow to test switching individual iterations to mkFit (like here a customization for detachedTripletStep can be given without enabling it with the `trackingMkFit` Modifierchain). This is intended only for short term.